go/test/endtoend/cluster/query_util.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"io"
+	"strings"
+
+	"vitess.io/vitess/go/sqltypes"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+// printQueryResult will pretty-print a QueryResult to the logger.
```

Comment:
nit: the comment should have the same case for the method name

Reply:
fixed